codehearts/remembear
# User insertion

## Description of changes

- Database functionality can be mocked
- New user data can be inserted into the sqlite database
- Running `cargo run` inserts a new user named `leland` into the database each time

## Reason for changes 

- Support user management and pave the way for providing new modules' data from the database

## Manual testing

1. Run `cargo run`
1. Run `sqlite3 remembear.sqlite3 '.dump users'` to see that user `leland` has been added with id `1`
1. Do this again and see another `leland` with id `2`
